src/stylesheets/layouts/_header.scss

```diff
@@ -2,44 +2,50 @@
   background-color: $color-mint;
   display: flex;
   justify-content: center;
+  position: relative;
 
   &__wrapper {
-    margin: 50px;
-    // position: relative;
+    max-width: 50rem;
+    margin: 5rem 6rem;
```

Comment:
you might want this to be your `position: relative` element, just so you don't have to put those `<img>` inside your `<h1>`